MetaMask/metamask-extension
# fix: UX: Multichain: Remove 'Connected Sites' global item when in MC

## **Description**

During QA @sleepytanya was getting tripped up by this legacy "Connected Sites" global menu item.  It should no longer be there once we ship Multichain, so I'm hiding it when the flag is on

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23789?quickstart=1)

## **Related issues**

Fixes: N/A

## **Manual testing steps**

1. Start extension with `MULTICHAIN=1` flag
2. Open the extension global three-dot menu
3. Don't see "Connected Sites"

## **Screenshots/Recordings**



### **Before**

N/A

### **After**

N/A

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
